airflow/operators/python.py

```diff
@@ -145,6 +145,69 @@ def execute_callable(self):
         return self.python_callable(*self.op_args, **self.op_kwargs)
 
 
+class PythonFunctionalOperator(PythonOperator):
+    """
+    Allows a workflow to "branch" or follow a path following the execution
```

Comment:
I am not sure about the documentation here :)

Reply:
Yup, ignore documentation XD

Reply:
Updated.